### ISSUE TITLE
Angular-cli V1.0.0 3rd party scripts or styles are not loading 

### ISSUE THREAD

**Issue by nguyendnhat** [angular/angular-cli#5944]
I'm using "@angular/cli": "^1.0.0"
And add material lite in to project 
"apps": [
    {
      "root": "Client",
      "outDir": "wwwroot/dist",
      "assets": [
        "assets",
        "favicon.ico"
      ],
      "index": "index.html",
      "main": "main.ts",
      "polyfills": "polyfills.ts",
      "test": "test.ts",
      "tsconfig": "tsconfig.app.json",
      "testTsconfig": "tsconfig.spec.json",
      "prefix": "CRM",
      "styles": [],
      "scripts": [ "../node_modules/material-design-lite/material.min.js" ]
    }
  ],

But it does not get loaded.
How fix it.

@Node: v6.10.2
@NPM: 3.10.10

**Comment by psykolm22** [comment]
```
  "apps": [{
        "styles": [
            "styles.css",
            "../node_modules/material-design-lite/material.min.css"
        ],
        "scripts": ["../node_modules/material-design-lite/material.min.js"],
```

Work for me with @angular-cli: 1.0.0 @Node: v6.10.0 @NPM: 3.10.10
Maybe you forgot to add  material.min.css  ?

**Comment by MHacker9404** [comment]
I'm also seeing this issue.  Created an angular-cli app from scratch with angular-cli 1.0,0, node:7.8.0.  In the json, i've got
    "scripts": [
        "../node_modules/jquery/dist/jquery.min.js"
      ],

and this is the component:

    import { Component, AfterViewInit } from '@angular/core';
    import * as jQuery from 'jquery';
    
    @Component({
      selector: 'app-root',
      templateUrl: './app.component.html',
      styleUrls: ['./app.component.css']
    })
    export class AppComponent implements AfterViewInit {
      title = 'app works!';
    
      ngAfterViewInit() {
        jQuery.delay(5000);
      }
    }

This is the error I get at runtime with "ng serve"
ERROR TypeError: __WEBPACK_IMPORTED_MODULE_1_jquery__.delay is not a function

**Comment by DavyDeDurpel** [comment]
I faced a similar issue.  In my case the css files were not loaded.
It took me all morning to figure out what was going wrong and after a long struggle I actually found the cause.

On my Windows laptop I have a directory containing a symbolic link to another directory.  My best guess is that webpack is not compatible with symbolic link directories on Windows.

An example:

full path to my project:  c:\Users\Public\Documents\angular-rocks
symbolic link path to my project: c:\data\angular-rocks

if I start my project from the full path then the css files are loaded.
If I start my project from the symbolic link path then the css files are not loaded.
